include/asterisk/app.h

```diff
@@ -1196,9 +1196,54 @@ struct ast_group_info;
 /*! \brief Split a group string into group and category, returning a default category if none is provided. */
 int ast_app_group_split_group(const char *data, char *group, int group_max, char *category, int category_max);
 
+/* \brief Create group meta if it doesn't exist  */
+int ast_app_group_add_meta_ifneeded(const char *group, const char *category);
```

Comment:
Does this need to be public? It seems to only be used in app.c?

Reply:
Fixed, now static.